### DISCUSSION TITLE
Spawn Codes for Stargates bringing them into the network/assigning custom addresses

### DISCUSSION THREAD

**Discussion by Temunai** [Povstalec/StargateJourney#25] [Q&A] [answered]
I thought I saw the answer to this a while ago but havnt been able to see it since. 
I was wondering how to spawn in one of the other gate designs and have it receive / set an address to it

**Comment by Povstalec** [comment, marked as answer]
`/give @p sgjourney:milky_way_stargate{BlockEntityTag:{ID:"-1-2-3-4-5-6-7-8-"}}`

This command suffix should work for every gate. The numbers can be effectively whatever you like them to be, but it must always be 8 numbers.
(Do note that it's better to use numbers larger than 35 if you want Universe Stargates to dial it)


`/give @p sgjourney:milky_way_stargate{BlockEntityTag:{PointOfOrigin:"sgjourney:kaliem", Symbols:"sgjourney:abydos"}}`

I don't have them written down separately, but here's a list of available [Points Of Origin](https://github.com/Povstalec/StargateJourney/tree/main/src/main/resources/data/sgjourney/sgjourney/point_of_origin) and [Symbols](https://github.com/Povstalec/StargateJourney/tree/main/src/main/resources/data/sgjourney/sgjourney/symbols)
(Also note that you won't see a difference in symbols used within the same galaxy if you don't have the Unique Symbols client config option turned on)

And in case someone is unsure, the individual parameters can be combined and separated by commas:
`/give @p sgjourney:milky_way_stargate{BlockEntityTag:{ID:"-1-2-3-4-5-6-7-8-", PointOfOrigin:"sgjourney:kaliem", Symbols:"sgjourney:abydos"}}`
